packages/SketchMorph2-Core.package/M2GripMorph.class/instance/mouseUp..st

```diff
@@ -4,5 +4,4 @@ mouseUp: evt
 		+ self position
 		- self cursorOffset
 		- self referencePoint.
-	self updatePosition.
```

Comment:
Maybe we should leave this line in? In case something goes wrong, it would fix itself when dragging the morph.